src/codeu/chat/client/core/View.java

```diff
@@ -117,23 +113,65 @@ public View(ConnectionSource source) {
   @Override
   public Collection<Message> getMessages(Collection<Uuid> ids) {
 
-    final Collection<Message> messages = new ArrayList<>();
+      final Collection<Message> messages = new ArrayList<>();
```

Comment:
Also the remaining of the file does not follow the indentation scheme, please fix all.